DataAccessLayer/ServiceInterfaces/IEmployeeService.cs

```diff
@@ -15,6 +15,19 @@ public interface IEmployeeService
         /// <summary>
         /// Method that takes in an email and password as parameters and returns an instance of the `Employee` class.
         /// </summary>
+        /// <param name="email"></param>
+        /// <param name="password"></param>
+        /// <returns></returns>
         Employee GetEmployee(string email, string password);
+
+        /// <summary>
+        /// Creates a new Employee record in the system with the provided details.
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="email"></param>
+        /// <param name="password"></param>
+        /// <param name="roleId"></param>
+        /// <returns></returns>
+        Employee PostEmployee(string name, string email, string password, int roleId);
```

Comment:
Be careful when you update some function on a class with an interface inheritance, because I think it will break because on DataAccessLayer/Services/EmployeeService.cs does not have PostEmployee function because you rename it.
Also, please add the comment information for each param comment